src/emulate.c

```diff
@@ -90,6 +90,19 @@ static void rv_exception_default_handler(riscv_t *rv)
 RV_EXCEPTION_LIST
 #undef _
 
+/* wrap load/store and insn misaligned handler */
```

Comment:
You shall explain the parameters in details.

Reply:
Updated